### PR TITLE
fix: fix resource card description height

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/cards/ResourceCard/style.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ const useStyles = makeStyles(() => ({
     flexDirection: 'column',
     justifyContent: 'space-between',
   },
+  description: {
+    height: '70px',
+    overflow: 'auto',
+  },
```
Comment (nborde-CSM) [inline src/cards/ResourceCard/style.js:16]: Is it not possible to take the opportunity to delete `style.js` and implement style directly inside `sx` balise ?

**Review by nborde-CSM — APPROVED** [review]
One question but LGTM :+1:

File: src/cards/ResourceCard/style.js (hunk 1 above)
Comment (csm-thu) [reply src/cards/ResourceCard/style.js:16]: IIRC the `sx` prop can only be used on _mui_ components, whereas we want to use it on a `div` here

**Pull request merged**